app/views/content/is-teaching-right-for-me/all-subjects.md

```diff
@@ -0,0 +1,299 @@
+
+## Philosophy 
+
+As a philosophy teacher you'll teach pupils ways of thinking about ethics, time and meaning. Help them explore the big questions of life and morality. 
+
+<a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=P1&visa_status=false">Search for philosophy teacher training courses</a>.
+
+## Physical education 
+
+As a P.E teacher you'll help pupils to develop motor skills and an interest for physical activity and fitness. 
+
+You can <a href="https://www.thenational.academy/teachers/programmes/physical-education-secondary-ks3-l/units">explore what teaching a P.E lesson could look like</a>.
+
+<a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=C6&visa_status=false">Search for physical education teacher training courses</a>.
+
+
+## Physical education with an EBacc subject
+
+You'll be able to teach P.E teacher and another subject from: English, maths, history, languages or geography.
+
+<a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=C7&visa_status=false">Search for physical education with an EBacc subject teacher training courses</a>.
+
+## Physics
+
+<a href="https://getintoteaching.education.gov.uk/funding-and-support/scholarships-and-bursaries?funding_widget%5Bsubject%5D=physics&button=#funding-widget">Tax-free scholarships of £30,000 and bursaries of £28,000 are available for eligible trainee physics teachers</a>. 
+
+<a href="/is-teaching-right-for-me/physics">You can learn more about teaching physics</a>. 
+
+<a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=F3&visa_status=false">Search for physics teacher training courses</a>.
+
+## Psychology 
+
+As a psychology teacher you'll help pupils develop an understanding of the ideas and values that characterise ‘self’ and others. 
+
+<a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=C8&visa_status=false">Search for psychology teacher training courses</a>.
+
+## Religious education 
+
+<a href="https://getintoteaching.education.gov.uk/funding-and-support/scholarships-and-bursaries?funding_widget%5Bsubject%5D=religious_group&button=#funding-widget">Tax-free bursaries of £10,000 are available for eligible trainee religious education teachers</a>. 
+
+As a religious education teacher you'll play a part in your pupils spiritual and moral development and prepare them for experiences of later life. 
```

Comment:
@caitlin-reidy pupils'

Reply:
@caitlin-reidy what sort of experiences are they preparing them for in later life? can we give any examples? And how will they help their spiritual and moral development - what sorts of things would they do?